pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/pip/PipFreezeOutputParser.java

```diff
@@ -15,11 +15,12 @@
  */
 package com.linkedin.gradle.python.util.pip;
 
+import org.gradle.api.GradleException;
```

Comment:
Another unnecessary change.

Reply:
This has been fixed I believe. (Only one import exists now).